freestyle/shared/src/test/scala/freestyle/implicits.scala

```diff
@@ -53,7 +53,7 @@ class implicitsTests extends WordSpec with Matchers {
 
     "provide a custom implicit not found message for a missing Capture instance" in {
       shapeless.test.illTyped(
-        """SCtors1[SCtors1.Op].x(1).exec[scala.util.Try]""",
+        """SCtors1[SCtors1.Op].x(1).freeS.exec[scala.util.Try]""",
```

Comment:
Just wondering, why do we need to lift to `FreeS`?
You can also call `exec` on `FreeS.Par` right?

Or am I missing what this test is about?